Adds flags to build a universal binary covering all current Apple architectures.

--- a/libjpeg/makefile.xc	2024-01-09 16:16:22
+++ b/libjpeg/makefile.xc	2025-12-30 18:46:48
@@ -22,7 +22,7 @@
 CC= cc
 
 # You may need to adjust these cc options:
-CFLAGS= -O2 -Wall
+CFLAGS= -O2 -Wall -arch x86_64 -arch arm64
 # Generally, we recommend defining any configuration symbols in jconfig.h,
 # NOT via -D switches here.
 
